x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -43,550 +43,649 @@ export enum AddDataTabs {
-              <EuiCodeBlock paddingSize="s" isCopyable>
-                {`wget -O pf-host-agent.tgz "https://artifacts.elastic.co/downloads/prodfiler/pf-host-agent-${stackVersion}-linux-arm64.tar.gz" && tar xzf pf-host-agent.tgz`}
-              </EuiCodeBlock>
-            </EuiText>
-          ),
+docker.elastic.co/observability/profiling-agent:${ stackVersion } /root/pf-host-agent \\
+-project-id=1 -secret-token=${ secretToken } \\
+-collection-agent=${ collectionAgentHost }` }
+                  </EuiCodeBlock>
+              ),
+            },
+          ],
         },
         {
-          title: i18n.translate('xpack.profiling.tabs.binaryGrantPermissionStep', {
-            defaultMessage: 'Grant executable permissions:',
+          key: AddDataTabs.Binary,
+          title: i18n.translate('xpack.profiling.tabs.binaryTitle', {
+            defaultMessage: 'Binary',
           }),
-          content: (
-            <EuiCodeBlock paddingSize="s" isCopyable>
-              chmod +x pf-host-agent/pf-host-agent
-            </EuiCodeBlock>
-          ),
-        },
-        {
-          title: i18n.translate('xpack.profiling.tabs.binaryRunHostAgentStep', {
-            defaultMessage: 'Run the Universal Profiling host-agent (requires root privileges):',
-          }),
-          content: (
-            <EuiCodeBlock paddingSize="s" isCopyable>
-              {`sudo pf-host-agent/pf-host-agent -project-id=1 -secret-token=${secretToken} -collection-agent=${collectionAgentHost}`}
-            </EuiCodeBlock>
-          ),
+          subTabs: ['x86_64', 'arm64'].map((arch: string) => {
+            const archi18nLabel = arch.toUpperCase().replaceAll('_','');
+            return {
+              key: arch,
+              title: `Linux ${arch}`,
+              // inside each sub tab you define the steps as usual
+              steps: [
+                {
+                  title: i18n.translate(`xpack.profiling.tabs.binaryDownloadStepFor${archi18nLabel}`, {
+                    defaultMessage: 'Download the binary:',
+                  }),
+                  content: (
+                      <EuiText>
+                        <EuiCodeBlock paddingSize="s" isCopyable>
+                          { `wget -O pf-host-agent.tgz "https://artifacts.elastic.co/downloads/prodfiler/pf-host-agent-${stackVersion}-linux-${arch}.tar.gz" && tar xzf pf-host-agent.tgz` }
+                        </EuiCodeBlock>
+                      </EuiText>
+                  ),
+                },
+                {
+                  title: i18n.translate(`xpack.profiling.tabs.binaryGrantPermissionStepFor${archi18nLabel}`, {
+                    defaultMessage: 'Grant executable permissions:',
+                  }),
+                  content: (
+                      <EuiCodeBlock paddingSize="s" isCopyable>
+                        { i18n.translate(
+                            'app_not_found_in_i18nrc.addDataView.chmodXPf-host-agentpf-host-agentCodeBlockLabel',
```

Comment:
@cauemarcondes: is the `app_not_found_in_i18nrc` expected?

Reply:
There's a new eslint rule that automatically create the i18n template for you. Please replace `app_not_found_in_i18nrc` by `xpack.profiling`...

I'll take a look how to fix it later.

Reply:
Thanks 🙏🏼